lua/kubectl/utils/string.lua

```diff
@@ -21,4 +21,12 @@ function M.divider(buf, char)
   vim.api.nvim_win_set_cursor(0, { line_count + 1, 0 })
 end
 
+function M.get_visual_selection()
+  local esc = vim.api.nvim_replace_termcodes("<esc>", true, false, true)
```

Comment:
I dislike how vim gets visual selection so much...

Reply:
Yes trust me, currently, there is no other way..